### PR TITLE
Add source code files for space travel

### PR DESCRIPTION
**Pull request by sebras**
I've done my best to enter the full source code of space travel as [scanned](https://www.tuhs.org/Archive/Distributions/Research/McIlroy_v0/12-92-119.pdf) and mentioned on the `pdp7-unix` mailing list.

N.B. This has been entered by hand, and I have no idea how to compile test this.

Also if I have done some mistake or placed the source code files at the wrong location. Feel free to fix it or let me know and I'll do it for you. :)

### CODE REVIEW

**Comment by sebras** [comment]
Only now did I see a [message](https://minnie.tuhs.org/pipermail/pdp7-unix/2019-October/000491.html) mentioning

> 12-92-119   st1-7.s   aap

 these files as being reserved... :-/ Sorry if I stepped on someone's foot. I'm happy to collaborate on these and compare editions. Unfortunately I'm still waiting for permission to be subscribed (and hence post) to `pdp7-unix`, so here is unfortunately my only way of communication at the moment.

File: scans/space-travel-1.s [hunk 1]
```diff
@@ -0,0 +1,248 @@
+! space travel 1
```
Comment (bsdimp) [inline scans/space-travel-1.s]: This is
```
" space travel 1
```
in the original

File: scans/space-travel-1.s [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+   dac aexp
+   fng
+   fad; fpone
+   fst; 3:0
+   lac 1b
+   tad d3
+   dac 1b
+   dac 2b
+   lac 3b
+   tad 3d
```
Comment (bsdimp) [inline scans/space-travel-1.s]: This is
```
tad d3
```
in the original

File: scans/space-travel-1.s [hunk 3]
```diff
@@ -0,0 +1,248 @@
...
+   fld; fpone
+   fst; sphi
+   fst; stheta
+   jms dspsca
+   fld; fpzero
+   fst; x
+   fst; ox
+   fst; cphi
+   fst; ctheta
+   law displist
```
Comment (bsdimp) [inline scans/space-travel-1.s]: page break
```
" 12-92-119.pdf page 2
```

File: scans/space-travel-1.s [hunk 4]
```diff
@@ -0,0 +1,248 @@
...
+   lac scale
+   cma
+   tad vscale
+   tad aexp
+   dac aexp
+   jms inscr
+      jmp loop4
+   tad o141577
+   dac i clistp
+   lac o164372
```
Comment (bsdimp) [inline scans/space-travel-1.s]: should note that the original 12-92-119.pdf page 2 has corrections here not reflected in this transcription.

File: scans/space-travel-1.s [hunk 5]
```diff
@@ -0,0 +1,248 @@
...
+   dac clistp i
+   cla
+   jms dsplanet
+loop4:
+   sys time "put delay here.....
+   dzm dispcl
```
Comment (bsdimp) [inline scans/space-travel-1.s]:
```
" Page 3 12-92-119.pdf
```

**Pull request merged**